en/routing.texy

```diff
@@ -198,17 +198,36 @@ class RouterFactory
 }
 \--
 
-.[note]
-The `@return` annotation is important and required for getting the code to work.
-
 **File app/config/config.neon**:
 
 /--neon
 services:
-	routerFactory: App\RouterFactory
-	router: @routerFactory::createRouter
+	router: App\Router\RouterFactory::createRouter
 \--
 
+.[caution]
+Order of routes is important, because it try from top to bottom.  Basic rule is we **declare routes from specific do general**.
+
+Count of routes has effect to speed of application, especially when generating links. It is therefore worthwhile to simplify the routing table.
```

Comment:
„has effect to speed of application“ -> „affects the speed [nebo možná spíš performance?] of application“